testssl.sh

```diff
@@ -14608,6 +14651,10 @@ parse_tls_serverhello() {
                                     258) dh_bits=4096 ; named_curve_str="ffdhe4096" ;;
                                     259) dh_bits=6144 ; named_curve_str="ffdhe6144" ;;
                                     260) dh_bits=8192 ; named_curve_str="ffdhe8192" ;;
+                                    4587) dh_bits=192 ; named_curve_str="SecP256r1MLKEM768" ;;
+                                    4588) dh_bits=192 ; named_curve_str="X25519MLKEM768" ;;
+                                    4589) dh_bits=256 ; named_curve_str="SecP384r1MLKEM1024" ;;
+                                    25497) dh_bits=128 ; named_curve_str="X25519Kyber768Draft00" ;;
```

Comment:
Why do you mark this one as 128 bits, but the MLKEM768 equivalent as 192 bits?  It's virtually the same algorithm.

Reply:
According to the NIST PQC security levels, ML-KEM-512, 768 and 1024 are equal in strength to AES-128, 192 and 256, respectively.
So this mapping assigns to each hybrid KEM the strength of the strongest component algorithm - except for Kyber768.

Reply:
https://pq-crystals.org/kyber/